storage/connect/tabmul.cpp

```diff
@@ -1367,10 +1377,10 @@ TDBDHR::TDBDHR(PTDBDHR tdbp) : TDBASE(tdbp)
   FileData = tdbp->FileData;
   Hsearch = tdbp->Hsearch;
   iFile = tdbp->iFile;
-  strcpy(Drive, tdbp->Drive);
-  strcpy(Direc, tdbp->Direc);
-  strcpy(Fname, tdbp->Fname);
-  strcpy(Ftype, tdbp->Ftype);
+  snprintf(Drive, sizeof(Drive), tdbp->Drive);
+  snprintf(Direc, sizeof(Direc), tdbp->Direc));
+  snprintf(Fname, sizeof(Fname), tdbp->Fname));
+  snprintf(Ftype, sizeof(Ftype), tdbp->Ftype);
```

Comment:
Why not something like `safe_strcpy()` here? I'm pretty sure doing this without an `"%s"` will trigger warnings on some compilers anyway.